test/group_A_case_1.yaml

```diff
@@ -0,0 +1,151 @@
+- method: SchedulePod
```

Comment:
reorganize to the following structure and rename group/case to meaningful name?
```
test/
└── config
    ├── group1
    │   ├── case1.yaml
    │   ├── case2.yaml
    │   └── config.yaml
    └── group2
        ├── case1.yaml
        └── config.yaml
```

Reply:
done.